12_quality_control.Rmd

```diff
@@ -293,6 +293,16 @@ plotColData(tse,"sum","SampleType", colour_by = "SampleType") +
     theme(axis.text.x = element_text(angle = 45, hjust=1))
 ```
 
+In addition, data can be rarefied with [subsampleCounts](https://microbiome.github.io/mia/reference/subsampleCounts.html), which normalises the samples to an equal number of reads. However, this practice is discouraged for the analysis of differentially abundant microorganisms (see [McMurdie PJ, Holmes S](https://doi.org/10.1371/journal.pcbi.1003531)).
```

Comment:
Add references via the @ mechanism. For instance citing OMA would work like [@OMA] (see intro.Rmd for an instance), and adding the corresponding entry to book.bib file.